cmake/BLTMacros.cmake

```diff
@@ -1557,6 +1557,7 @@ endmacro()
 ##------------------------------------------------------------------------------
 ## blt_check_code_compiles(CODE_COMPILES <variable>
 ##                         VERBOSE_OUTPUT <ON|OFF (default OFF)>
+##                         LINK_LIBRARIES <libs>
```

Comment:
We should actually rename this parameter to be `DEPENDS_ON` to match `blt_add_library` and `blt_add_executable`.

Reply:
Done